v1/subscriber/subscriber.go

```diff
@@ -300,14 +286,14 @@ func (p *API) GetClientIDBySubID(subID string) (clientIDs []uuid.UUID) {
 	return clientIDs
 }
 
-// GetClientIDAddressByResource  get  subscriptionOne information
+// GetClientIDAddressByResource get subscriptionOne information
 func (p *API) GetClientIDAddressByResource(resource string) map[uuid.UUID]*types.URI {
 	clients := map[uuid.UUID]*types.URI{}
 	p.SubscriberStore.RLock()
 	defer p.SubscriberStore.RUnlock()
 	for _, subs := range p.SubscriberStore.Store {
 		for _, sub := range subs.SubStore.Store {
-			if sub.GetResource() == resource {
+			if strings.Contains(sub.GetResource(), resource) {
```

Comment:
Could this potentially create a situation where a wrong subscription is associated with the client, for example if there are two resources - "abcd" and "xyabcdz"?